utils/python/hsfs_utils.py

```diff
@@ -304,6 +305,9 @@ def parse_isoformat_date(da: str) -> datetime:
             run_feature_monitoring(job_conf)
 
         success = True
+    except Exception:
+        # Printing stack trace of exception so that logs are not lost.
+        print(traceback.format_exc())
```

Comment:
I have never used the traceback module. This was tested manually right?

Reply:
Yes I have tested it manually. I have attached the screenshot of the logs printed after the fix. Source :https://stackoverflow.com/questions/3702675/catch-and-print-full-python-exception-traceback-without-halting-exiting-the-prog
<img width="2020" alt="image" src="https://github.com/user-attachments/assets/0c9c9c39-03bc-4463-8de9-446d57d08931">